tests/test_sanity.py

```
@@ -0,0 +1,8 @@
+def test_environment_sanity():
+    assert True
+
+def test_import_sanity():
+    try:
+        import insomnia_run
+    except ImportError:
+        assert False, "insomnia_run package not found. Did you run 'pip install -e .'?"
```

Ruff (0.14.10)

Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)